ThriftHandler/DBHandler.cpp

```diff
@@ -491,10 +499,11 @@ void DBHandler::krb5_connect(TKrb5Session& session,
   THROW_DB_EXCEPTION("Unauthrorized Access. Kerberos login not supported");
 }
 
-void DBHandler::connect(TSessionId& session,
+void DBHandler::connect(TSessionId& session_id,
                         const std::string& username,
                         const std::string& passwd,
                         const std::string& dbname) {
+  logger::set_new_request_id();
   auto stdlog = STDLOG();  // session_info set by connect_impl()
```

Comment:
Comment in `internal` is `// session set by connect_impl()`.
(Should probably be `session_id` but we should fix it upstream first.)